src/System.Net.Primitives/src/System/Net/CookieContainer.cs

```diff
@@ -983,15 +991,16 @@ public void SetCookies(Uri uri, string cookieHeader)
     }
 
     [Serializable]
+    [System.Runtime.CompilerServices.TypeForwardedFrom("System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
     internal struct PathList
```

Comment:
We can change that later. Or even better, if you want to help you could spin up a PR which optimizes my recent changes without killing the just introduced serialization support.